dashboard/src/shared/KubeappsGrpcClient.ts

```diff
@@ -0,0 +1,81 @@
+import { grpc } from "@improbable-eng/grpc-web";
+import { PackagesServiceClientImpl } from "gen/kubeappsapis/core/packages/v1alpha1/packages";
+import {
+  GrpcWebImpl,
+  PluginsServiceClientImpl,
+} from "gen/kubeappsapis/core/plugins/v1alpha1/plugins";
+import { FluxV2PackagesServiceClientImpl } from "gen/kubeappsapis/plugins/fluxv2/packages/v1alpha1/fluxv2";
+import { HelmPackagesServiceClientImpl } from "gen/kubeappsapis/plugins/helm/packages/v1alpha1/helm";
+import { KappControllerPackagesServiceClientImpl } from "gen/kubeappsapis/plugins/kapp_controller/packages/v1alpha1/kapp_controller";
+import { Auth } from "./Auth";
+import * as URL from "./url";
+
+export class KubeappsGrpcClient {
+  private grpcWebImpl!: GrpcWebImpl;
+  private transport: grpc.TransportFactory;
+
+  // core apis
+  private packagesServiceClientImpl!: PackagesServiceClientImpl;
+  private pluginsServiceClientImpl!: PluginsServiceClientImpl;
+
+  // plugins package apis
+  private helmPackagesServiceClientImpl!: HelmPackagesServiceClientImpl;
+  private kappControllerPackagesServiceClientImpl!: KappControllerPackagesServiceClientImpl;
+  private fluxv2PackagesServiceClientImpl!: FluxV2PackagesServiceClientImpl;
+
+  constructor(transport?: grpc.TransportFactory) {
+    this.transport = transport ?? grpc.CrossBrowserHttpTransport({ withCredentials: true });
```

Comment:
What was the end result of `withCredentials`? I thought you'd tested earlier and it was required as true only if you wanted creds to be sent across domains?

Reply:
Oops, sorry! It was committed by mistake (perhaps when merging branches?). It is certainly not required; fixing right away.